### ISSUE TITLE
MSCTF.dll cause crash application (not launch)

### ISSUE THREAD

**Issue by batressc** [microsoft/terminal#1737]
# Environment

```none
Windows build number: [Versión 10.0.18362.207]
Windows Terminal version (if applicable): 0.2.1715.0
```

# Steps to reproduce

1. Download windows terminal from Microsoft Store
2. Try execute application
3. Don't run, only crash

# Expected behavior

Execute application

# Actual behavior

When y try execute Windows terminal only show the border of application and then crash. I get the dump from Event Viewer and the problem is the module MSCTF.dll. The dump is this:

![image](https://user-images.githubusercontent.com/9021791/60401214-4b499500-9b3b-11e9-8018-fad5d2c4dfc4.png)

**Comment by DHowett-MSFT** [comment]
Can you find an event from "Windows Error Reporting" for WindowsTerminal.exe and share that one as well? Thank you.

**Comment by batressc** [comment]
This is the report (you can donwload the report file from [this link](https://rushfiles.one/client/publiclink.aspx?id=0mbCNRThSH))

﻿Version=1
EventType=MoBEX
EventTime=132063955245545698
ReportType=2
Consent=1
UploadTime=132063955250435704
ReportStatus=268566528
ReportIdentifier=12bf1a26-2bdb-4f83-ae5e-bdb994b6f342
IntegratorReportIdentifier=e7510e17-436e-4730-a1a7-831a9030f8d3
Wow64Host=34404
NsAppName=praid:App
AppSessionGuid=00003bc0-0002-0003-1e80-6eb3772fd501
TargetAppId=U:Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe!App
TargetAppVer=0.2.1715.0_x64_!2019//06//20:23:18:49!0!WindowsTerminal.exe
BootId=4294967295
ServiceSplit=152896512
TargetAsId=2416
UserImpactVector=808452912
IsFatal=1
EtwNonCollectReason=4
Response.BucketId=77f0e770b3ebf668ad81507b45818b9b
Response.BucketTable=5
Response.LegacyBucketId=2126068989472902043
Response.type=4
Sig[0].Name=Nombre completo del paquete
Sig[0].Value=Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe
Sig[1].Name=Nombre de la aplicación
Sig[1].Value=praid:App
Sig[2].Name=Versión de la aplicación
Sig[2].Value=1.0.1906.20005
Sig[3].Name=Marca de tiempo de la aplicación
Sig[3].Value=5d0c1459
Sig[4].Name=Nombre del módulo con errores
Sig[4].Value=MSCTF.dll
Sig[5].Name=Versión del módulo con errores
Sig[5].Value=10.0.18362.207
Sig[6].Name=Marca de tiempo del módulo con errores
Sig[6].Value=e8756a1a
Sig[7].Name=Desplazamiento de excepción
Sig[7].Value=00000000000f31a0
Sig[8].Name=Código de excepción
Sig[8].Value=c0000005
Sig[9].Name=Datos de excepción
Sig[9].Value=0000000000000008
DynamicSig[1].Name=Versión del sistema operativo
DynamicSig[1].Value=10.0.18362.2.0.0.256.48
DynamicSig[2].Name=Id. de configuración regional
DynamicSig[2].Value=17418
DynamicSig[22].Name=Información adicional 1
DynamicSig[22].Value=2f23
DynamicSig[23].Name=Información adicional 2
DynamicSig[23].Value=2f2387fbf6f05e0f9bce10bc586c9d6c
DynamicSig[24].Name=Información adicional 3
DynamicSig[24].Value=ee5b
DynamicSig[25].Name=Información adicional 4
DynamicSig[25].Value=ee5be4fda7ee663b7d8d879efc2b3899
UI[2]=C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe\WindowsTerminal.exe
LoadedModule[0]=C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe\WindowsTerminal.exe
LoadedModule[1]=C:\WINDOWS\SYSTEM32\ntdll.dll
LoadedModule[2]=C:\Program Files\AVAST Software\Avast\aswhook.dll
LoadedModule[3]=C:\WINDOWS\System32\KERNEL32.DLL
LoadedModule[4]=C:\WINDOWS\System32\KERNELBASE.dll
LoadedModule[5]=C:\WINDOWS\System32\USER32.dll
LoadedModule[6]=C:\WINDOWS\System32\win32u.dll
LoadedModule[7]=C:\WINDOWS\System32\GDI32.dll
LoadedModule[8]=C:\WINDOWS\System32\gdi32full.dll
LoadedModule[9]=C:\WINDOWS\System32\msvcp_win.dll
LoadedModule[10]=C:\WINDOWS\System32\ucrtbase.dll
LoadedModule[11]=C:\WINDOWS\System32\shcore.dll
LoadedModule[12]=C:\WINDOWS\System32\msvcrt.dll
LoadedModule[13]=C:\WINDOWS\System32\RPCRT4.dll
LoadedModule[14]=C:\WINDOWS\System32\combase.dll
LoadedModule[15]=C:\WINDOWS\System32\bcryptPrimitives.dll
LoadedModule[16]=C:\WINDOWS\System32\ole32.dll
LoadedModule[17]=C:\WINDOWS\System32\advapi32.dll
LoadedModule[18]=C:\WINDOWS\System32\sechost.dll
LoadedModule[19]=C:\WINDOWS\System32\OLEAUT32.dll
LoadedModule[20]=C:\WINDOWS\SYSTEM32\dwmapi.dll
LoadedModule[21]=C:\Program Files\WindowsApps\Microsoft.VCLibs.140.00.UWPDesktop_14.0.27629.0_x64__8wekyb3d8bbwe\MSVCP140.dll
LoadedModule[22]=C:\Program Files\WindowsApps\Microsoft.VCLibs.140.00.UWPDesktop_14.0.27629.0_x64__8wekyb3d8bbwe\VCRUNTIME140.dll
LoadedModule[23]=C:\WINDOWS\System32\IMM32.DLL
LoadedModule[24]=C:\WINDOWS\System32\kernel.appcore.dll
LoadedModule[25]=C:\WINDOWS\System32\clbcatq.dll
LoadedModule[26]=C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe\TerminalApp.dll
LoadedModule[27]=C:\WINDOWS\System32\SHELL32.dll
LoadedModule[28]=C:\WINDOWS\System32\cfgmgr32.dll
LoadedModule[29]=C:\WINDOWS\System32\windows.storage.dll
LoadedModule[30]=C:\WINDOWS\System32\profapi.dll
LoadedModule[31]=C:\WINDOWS\System32\powrprof.dll
LoadedModule[32]=C:\WINDOWS\System32\UMPDC.dll
LoadedModule[33]=C:\WINDOWS\System32\shlwapi.dll
LoadedModule[34]=C:\WINDOWS\System32\cryptsp.dll
LoadedModule[35]=C:\WINDOWS\System32\bcrypt.dll
LoadedModule[36]=C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe\Microsoft.UI.Xaml.Markup.dll
LoadedModule[37]=C:\WINDOWS\SYSTEM32\twinapi.appcore.dll
LoadedModule[38]=C:\WINDOWS\SYSTEM32\RMCLIENT.dll
LoadedModule[39]=C:\WINDOWS\SYSTEM32\threadpoolwinrt.dll
LoadedModule[40]=C:\Windows\System32\Windows.UI.Xaml.dll
LoadedModule[41]=C:\Windows\System32\CoreMessaging.dll
LoadedModule[42]=C:\Windows\System32\Bcp47Langs.dll
LoadedModule[43]=C:\Windows\System32\iertutil.dll
LoadedModule[44]=C:\Windows\System32\dcomp.dll
LoadedModule[45]=C:\Windows\System32\WinTypes.dll
LoadedModule[46]=C:\Windows\System32\OneCoreUAPCommonProxyStub.dll
LoadedModule[47]=C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe\TerminalSettings.dll
LoadedModule[48]=C:\WINDOWS\system32\uxtheme.dll
LoadedModule[49]=C:\WINDOWS\System32\MSCTF.dll
LoadedModule[50]=C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe\TerminalControl.dll
LoadedModule[51]=C:\WINDOWS\SYSTEM32\d2d1.dll
LoadedModule[52]=C:\WINDOWS\SYSTEM32\DWrite.dll
LoadedModule[53]=C:\WINDOWS\SYSTEM32\dxgi.dll
LoadedModule[54]=C:\WINDOWS\SYSTEM32\d3d11.dll
LoadedModule[55]=C:\WINDOWS\SYSTEM32\dxcore.dll
LoadedModule[56]=C:\WINDOWS\SYSTEM32\TextInputFramework.dll
LoadedModule[57]=C:\WINDOWS\SYSTEM32\CoreUIComponents.dll
LoadedModule[58]=C:\WINDOWS\SYSTEM32\ntmarta.dll
LoadedModule[59]=C:\Windows\System32\Windows.UI.dll
LoadedModule[60]=C:\Windows\System32\InputHost.dll
LoadedModule[61]=C:\Windows\System32\PROPSYS.dll
LoadedModule[62]=C:\Windows\System32\UiaManager.dll
LoadedModule[63]=C:\WINDOWS\SYSTEM32\urlmon.dll
LoadedModule[64]=C:\Windows\System32\CRYPTBASE.DLL
LoadedModule[65]=C:\Windows\System32\MrmCoreR.dll
LoadedModule[66]=C:\WINDOWS\SYSTEM32\windows.staterepositorycore.dll
LoadedModule[67]=C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_c1a085cc86772d3f\nvldumdx.dll
LoadedModule[68]=C:\WINDOWS\SYSTEM32\VERSION.dll
LoadedModule[69]=C:\Windows\System32\bcp47mrm.dll
LoadedModule[70]=C:\WINDOWS\System32\crypt32.dll
LoadedModule[71]=C:\WINDOWS\System32\MSASN1.dll
LoadedModule[72]=C:\WINDOWS\System32\WINTRUST.DLL
LoadedModule[73]=C:\WINDOWS\System32\imagehlp.dll
LoadedModule[74]=C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe\Microsoft.UI.Xaml.dll
LoadedModule[75]=C:\Program Files\WindowsApps\Microsoft.VCLibs.140.00_14.0.27323.0_x64__8wekyb3d8bbwe\MSVCP140_APP.dll
LoadedModule[76]=C:\Program Files\WindowsApps\Microsoft.VCLibs.140.00_14.0.27323.0_x64__8wekyb3d8bbwe\VCRUNTIME140_APP.dll
LoadedModule[77]=C:\Windows\System32\Windows.UI.Immersive.dll
LoadedModule[78]=C:\WINDOWS\system32\DataExchange.dll
LoadedModule[79]=C:\WINDOWS\system32\rsaenh.dll
LoadedModule[80]=C:\WINDOWS\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_c1a085cc86772d3f\nvwgf2umx.dll
LoadedModule[81]=C:\WINDOWS\SYSTEM32\WINMM.dll
LoadedModule[82]=C:\WINDOWS\SYSTEM32\WINMMBASE.dll
LoadedModule[83]=C:\WINDOWS\system32\nvspcap64.dll
LoadedModule[84]=C:\WINDOWS\SYSTEM32\uiautomationcore.dll
LoadedModule[85]=C:\WINDOWS\SYSTEM32\sxs.dll
LoadedModule[86]=C:\Windows\System32\Windows.UI.Xaml.Controls.dll
LoadedModule[87]=C:\WINDOWS\system32\windowscodecs.dll
LoadedModule[88]=C:\Windows\System32\Windows.Globalization.dll
LoadedModule[89]=C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe\TerminalConnection.dll
LoadedModule[90]=C:\Windows\System32\Windows.ApplicationModel.dll
LoadedModule[91]=C:\Windows\System32\Windows.Energy.dll
LoadedModule[92]=C:\Windows\System32\Windows.Graphics.dll
LoadedModule[93]=C:\WINDOWS\system32\directmanipulation.dll
LoadedModule[94]=C:\Windows\System32\wuceffects.dll
LoadedModule[95]=C:\WINDOWS\SYSTEM32\daxexec.dll
LoadedModule[96]=C:\WINDOWS\SYSTEM32\FLTLIB.DLL
LoadedModule[97]=C:\WINDOWS\SYSTEM32\container.dll
LoadedModule[98]=C:\WINDOWS\SYSTEM32\IPHLPAPI.DLL
LoadedModule[99]=C:\WINDOWS\SYSTEM32\capauthz.dll
State[0].Key=Transport.DoneStage1
State[0].Value=1
OsInfo[0].Key=vermaj
OsInfo[0].Value=10
OsInfo[1].Key=vermin
OsInfo[1].Value=0
OsInfo[2].Key=verbld
OsInfo[2].Value=18362
OsInfo[3].Key=ubr
OsInfo[3].Value=207
OsInfo[4].Key=versp
OsInfo[4].Value=0
OsInfo[5].Key=arch
OsInfo[5].Value=9
OsInfo[6].Key=lcid
OsInfo[6].Value=3082
OsInfo[7].Key=geoid
OsInfo[7].Value=217
OsInfo[8].Key=sku
OsInfo[8].Value=48
OsInfo[9].Key=domain
OsInfo[9].Value=0
OsInfo[10].Key=prodsuite
OsInfo[10].Value=256
OsInfo[11].Key=ntprodtype
OsInfo[11].Value=1
OsInfo[12].Key=platid
OsInfo[12].Value=10
OsInfo[13].Key=sr
OsInfo[13].Value=0
OsInfo[14].Key=tmsi
OsInfo[14].Value=1109
OsInfo[15].Key=osinsty
OsInfo[15].Value=3
OsInfo[16].Key=iever
OsInfo[16].Value=11.175.18362.0-11.0.130
OsInfo[17].Key=portos
OsInfo[17].Value=0
OsInfo[18].Key=ram
OsInfo[18].Value=16328
OsInfo[19].Key=svolsz
OsInfo[19].Value=464
OsInfo[20].Key=wimbt
OsInfo[20].Value=0
OsInfo[21].Key=blddt
OsInfo[21].Value=190318
OsInfo[22].Key=bldtm
OsInfo[22].Value=1202
OsInfo[23].Key=bldbrch
OsInfo[23].Value=19h1_release
OsInfo[24].Key=bldchk
OsInfo[24].Value=0
OsInfo[25].Key=wpvermaj
OsInfo[25].Value=0
OsInfo[26].Key=wpvermin
OsInfo[26].Value=0
OsInfo[27].Key=wpbuildmaj
OsInfo[27].Value=0
OsInfo[28].Key=wpbuildmin
OsInfo[28].Value=0
OsInfo[29].Key=osver
OsInfo[29].Value=10.0.18362.207.amd64fre.19h1_release.190318-1202
OsInfo[30].Key=buildflightid
OsInfo[30].Value=c9147e9f-d925-4671-aa33-cb8d83e5df04
OsInfo[31].Key=edition
OsInfo[31].Value=Professional
OsInfo[32].Key=ring
OsInfo[32].Value=Retail
OsInfo[33].Key=expid
OsInfo[34].Key=containerid
OsInfo[35].Key=containertype
OsInfo[36].Key=edu
OsInfo[36].Value=0
FriendlyEventName=Dejó de funcionar
ConsentKey=MoBEX
AppName=WindowsTerminal.exe
AppPath=C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_0.2.1715.0_x64__8wekyb3d8bbwe\WindowsTerminal.exe
NsPartner=windows
NsGroup=windows8
ApplicationIdentity=8461193447C22A48F6CEB505033A2CFE
MetadataHash=1903994276

**Comment by DHowett-MSFT** [comment]
Hey, we think this was fixed with 0.2.1813.0 (the early July servicing release). Please feel free to reopen this if you still have the issue.
